src/test/java/resources/identityMapperTest.feature

```diff
@@ -39,4 +40,131 @@ Feature: Identity Account Mapper Api Test
     When I call the account lookup API 10 times with expected status of 202 and stub "/accountLookup"
     And I make the "PUT" request to "/accountLookup" endpoint with expected status of 200
     Then I should be able to verify that the "PUT" method to "/accountLookup" endpoint received 10 request
+
+
+  Scenario: Update a single beneficiary with Payment Modality - Mobile Money
+    When I create an IdentityMapperDTO for 1 Register Beneficiary with payment modality as "ACCOUNT_ID"
+    When I call the register beneficiary API with expected status of 202 and stub "/registerBeneficiaryApiTest"
+    When I make the "PUT" request to "/registerBeneficiaryApiTest" endpoint with expected status of 200
+    When I create an IdentityMapperDTO for 1 update Beneficiary with payment modality as "MSISDN"
+    When I call the update beneficiary API with expected status of 202 and stub "/updateBeneficiaryApiTest"
+    When I make the "PUT" request to "/updateBeneficiaryApiTest" endpoint with expected status of 200
```

Comment:
what does this line has to to in the test.
When I make the "PUT" request to "/updateBeneficiaryApiTest" endpoint with expected status of 200

Reply:
I am not sure about this but as per my understanding this might be required to fetch the callback from stub